enterprise/server/cmd/ci_runner/main.go

```diff
@@ -235,7 +235,7 @@ func (r *buildEventReporter) Start(startTime time.Time) error {
 
 	options := []string{}
 	if *besBackend != "" {
-		options = append(options, fmt.Sprintf("--bes_backend=%s'", *besBackend))
+		options = append(options, fmt.Sprintf("--bes_backend=%s", *besBackend))
```

Comment:
I think we want to wrap the `%s` in single-quotes, similar to the `--remote_header` arg below